.travis.yml

```diff
@@ -1,45 +1,46 @@
 # Travis CI Configuration File
 
+# Tell Travis CI which distro to use
+dist: trusty
+
 # Tell Travis CI we're using PHP
 language: php
 
-# Tell Travis CI which distro to use
-dist: trusty
+# Run Matrix for these PHP versions
+php:
+- "5.6"
+- "7.0"
+- "7.2"
+#- "7.0"
+#  env: "SWITCH_TO_PHP=5.2"
+#  dist: precise
```

Comment:
Noting that these should probably be enabled at some point again as they're part of the initial setup:
```
#- "7.0"
#  env: "SWITCH_TO_PHP=5.2"
#  dist: precise
```

...I'm just not sure what exactly happens there. 🙃

Reply:
Yep, gotcha. Looking into that now.